src/core/Akka/Pattern/CircuitBreakerState.cs

```diff
@@ -37,9 +37,9 @@ public Open(CircuitBreaker breaker)
         /// <param name="body">N/A</param>
         /// <exception cref="OpenCircuitException">This exception is thrown automatically since the circuit is open.</exception>
         /// <returns>N/A</returns>
-        public override async Task<T> Invoke<T>(Func<Task<T>> body)
```

Comment:
Watch out, @Horusiath introduced these as part of #3754 , though I've been willing to ask how that works actually.

Reply:
I'll need to check references to see if this actually gets used anywhere.

Reply:
Its fine, the async keyword is just a syntactic sugar, it just flags the compiler that the function uses await and it should optimize it to use them. Since the method doesn't use await, we don't need one.

Reply:
By the way, you can use the async version by passing a `Func` that produces a task into a circuit breaker like this:
```
await _circuitBreaker.WithCircuitBreaker(() => MyTask);
```
currently, it accepts `Func<Task>` and `Func<Task<T>`